### PR TITLE
DOC: small formatting fixes

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
I've just noticed that the releases had not been added to the front page. This PR adds them.

The PR also fixes minor problems fix the documentation.

### CODE REVIEW

**Comment by kchawla-pi** [comment]
Cool! Thanks Gael.

Would you add a `Known Issues` section to the `whats_new.rst`, below the `Fixes` section for `0.5.1`?
I drafted the text in a local branch, you can fix this and add it?

```rst
Known Issues
------------

- On Python2, ``plotting.view_connectome()`` & ``plotting.view_markers()``
  do not show parameters names in function signature
  when using help() and similar features.
  Please refer to the docstring for this information.

``` 
Also, I miss typed `Contributors` for `0.5.1`, please fix that as well?

**Comment by kchawla-pi** [comment]
Hey, I will push the followup of my comments to this branch and merge this soon.
If you don't want me to do that let me know soon.

I am testing #1969 and in case of problems would like to do a clean revert, not have another merge after mine potentially complicating the revert.

**Comment by kchawla-pi** [comment]
I pushed he changes but they are not showing up here. Must have done it incorrectly. Did you receive them on your origin?

**Comment by GaelVaroquaux** [comment]
All is fine with me. Thanks!

⁣Sent from my phone. Please forgive typos and briefness.​

File: doc/whats_new.rst [hunk 1]
```diff
@@ -22,22 +22,22 @@ Changes
...
-- plotting.view_connectome parameter names are consistent with plot_connectome:
+- :func:`nilearn.plotting.view_connectome` default colormap is `bwr`, consistent with plot_connectome.
+- :func:`nilearn.plotting.view_connectome` parameter names are consistent with plot_connectome:
 
- - coords is now node_coord
- - marker_size is noe node_size
- - cmap is now edge_cmap
- - threshold is now edge_threshold
+  - coords is now node_coord
+  - marker_size is noe node_size
```
Comment (bthirion) [inline doc/whats_new.rst]: now

**Pull request merged**